sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/pythonLogicalOperators.scala

```diff
@@ -274,6 +274,24 @@ case class ArrowEvalPythonUDTF(
     copy(child = newChild)
 }
 
+/**
+ * A logical plan that evaluates the 'analyze' method of a [[PythonUDTF]] on the executors.
```

Comment:
Maybe this is beyond the scope of this PR, but could we add more comments/examples here on how a polymorphic UDTF can be transformed into a plan with this node where it's `analyze` method can be evaluated on executors?

Reply:
Sure, I added some more comments here to help explain.